### DISCUSSION TITLE
仿生人会摊电子饼吗？📱

### DISCUSSION THREAD

**Discussion by King-of-Infinite-Space** [King-of-Infinite-Space/thoughts#174] [博文]
不知道。但是人类会。很会。而且这饼越摊越大。饼是什么馅的？有很多种，比如苹果、三鲜、小米……

两年多以前我说过
> 私以为如今许多手机厂商已经陷入了误区，一味追求尺寸和屏占比。“Size matters” doesn’t mean “the bigger, the better”。这件商品叫做“手机”，是要拿在手上用的，所以在决定屏幕和机身尺寸时首要考虑的应该是握着手中时操作是否方便。

然而这两年来「做大」的风气越刮越盛，许多厂商已接近走火入魔了。市面上的手机看似名目繁多，可是能**单手握持使用**的越来越少。

我在手机数据网站GSMArena用以下条件进行了搜索
- 价格€150以上、屏幕4.5寸以上
  （排除小型低端机和智能手表）
- **宽度不超过72mm**
(作为参考，一个对角线长6寸、长宽比为19.5:9的矩形宽度为64mm。6.1寸的iPhone 12的宽度为71.5mm。)

我按年份进行搜索，并比较了有无宽度限制条件下搜索结果的数量。近几年的结果如下——
2017年，42 / 226 ≈ 19% （前「全面屏」时代）
2018年，34 / 249 ≈ 14%
2019年，28 / 271 ≈ 10%
2020年， 19 / 341 ≈ 6%

新产品越来越多，但符合条件的越来越少。难道人类的手在逐年变大？两三年前6±0.2寸是相当普遍的尺寸，而今这种「小屏」手机（事实上完全不小）却变成了边缘市场。这是因为需求过于小众呢，还是因为消费者根本没得选呢？

（注：上面只是粗略统计。分母的增加也可能是因为手机变贵了。另外，如果能获得相关数据，统计一下价格、尺寸、重量等特征的分布会有点意思。）

[2020年的19个搜索结果](https://www.gsmarena.com/results.php3?nYearMin=2020&nYearMax=2020&nPriceMin=150&nWidthMax=72&fDisplayInchesMin=4.5)（其中有两个重复，实为17个）里面主要有哪些公司的产品？苹果、三星、索尼、谷歌等。中国品牌呢？独苗华为P40。其他国产厂商尽管搞得如火如荼，但都在摊大饼，全然不顾这部分需求，还说什么没有市场。两大行业巨头果星都在此列，它们不懂市场吗？

其实「做大」的原因不难理解。各厂商努力给手机加入更多功能、使用更好的硬件，这些都需要空间。而为了在增强性能的同时不牺牲续航，只好增大电池的容量。于是曾经机灵的「手机」变得大而重，最终成功地否定了自身。当厨师努力地往饺子里塞进更多的馅，饺子的个头越来越大，最后就变成了包子。当「手机」不能单手握持、必须两手捧着使用时，它们应该被叫作「掰机」。

这种关于手机的原教旨主义值得坚持吗？至少人类的手就这么大，单手操作的需求总是存在的。但显然主流市场走向了不同的方向。或许刚刚提到的手机的自我否定正说明了它的进化。我们可以暂时跳出「手机」这个名字，重新审视这些和人们形影不离的东西。毕竟，尽管承袭了名字，现在的所谓智能手机和二十年前的手机早已不是同一物种。不妨将镜头拉远，回顾一下过去。

<img src='https://user-images.githubusercontent.com/27502578/105127930-e6a0b300-5aaf-11eb-8ab3-204a232dca4e.png' width=350>

2007年发布的初代iPhone开创了一个时代。当时的发布会上，乔布斯[这样宣布](https://singjupost.com/steve-jobs-iphone-2007-presentation-full-transcript/?singlepage=1)——
> Well, today we’re introducing three revolutionary products of this class. The first one is a widescreen iPod with touch controls. The second is a revolutionary mobile phone. And the third is a breakthrough Internet communications device.
> 
> So, three things: a widescreen iPod with touch controls; a revolutionary mobile phone; and a breakthrough Internet communications device. An iPod, a phone, and an Internet communicator. An iPod, a phone… are you getting it? These are not three separate devices, this is one device, and we are calling it iPhone. Today, Apple is going to reinvent the phone, and here it is.

尽管这话术有着典型果味，他的确道出了这类设备的本质。那之前的手机只是通讯设备，而那之后的手机是一种便携式联网通用计算设备（兼通讯设备）。通讯似乎已经只是遗传下来的原始特征，而真正的关键在于「通用」和「便携」。「通用」是说在诸多日用电子设备中手机的集成度最高、功能最全面。若单论某一功能（除了通话），总有其他设备比手机做得好。但手机的优势在于它什么都能做。这一点本身并不稀奇，但当「通用」和「便携」结合起来，它就威力倍增，让人拿得起放不下了。

关于那场发布会还有一个有趣的细节。乔布斯在介绍革新了人机交互方式的触屏时说
> What we’re going to do is get rid of all these buttons and just make a giant screen. A giant screen.

哇，一块巨大的屏幕。有多大？

3.5寸。

当然，以当年的标准这确实算大了。不过事后来看，他真的一语成谶……

现在的全面屏手机，看上去就是一块巨大的屏幕。它们变得越来越大，是为了增强通用性而在便携性上做了妥协。这大概是*技术思维*的产物。尽管技术常被视为推动社会进步的力量，但它推进的方向真的是人们希望的吗？将最新的技术成果拿在手上把玩或许令人欣喜，但那些新特性有多少是商业竞争催生的，又有多少是人们真正需要的？手机年年推陈出新，但往往人们拿来做同样的事。未老先衰的旧设备被打入冷宫，悲惨地沦为电子垃圾。在技术已经深刻改变了人们的生活形态的年代，*以人为本*的视角变得尤其重要。

前文提到的单手操作，正是从人的需求出发的一个例子。尽管手机可以做很多事情，这并不意味着这些事情都应该用手机来做。厂商可以开发大而全的手机，但人们也需要小而精的产品。而在如今的市场上，消费者无从选择。

前面我搜索数据时划定的标准是72mm，但考虑到如今手机屏幕偏长，这个尺寸已经让人难以单手控制了。我有着中等大小的男性人类手，手机宽度为70±1mm，在单手握持姿势不变的情况下是无法用拇指触摸屏幕上角的。想必手小的人比如女性或少年儿童就更难了。如果按照前文的条件并把宽度进一步限制在68mm以内，2020年上市的那三百多款手机里符合要求的只有3款——iPhone SE，iPhone 12 mini，以及一款长条索尼。

宽67.3mm的SE样式源自iPhone 6/7/8，尺寸也基本相同。多年来人们习惯的，自然就是不错的。新产品12 mini的宽度只有64.2mm，在这个时代实属极其稀有的袖珍动物。希望它不要灭绝。至少在这一方面，苹果做了一件**人**事，没有被技术思维带入歧途。非常遗憾其他厂商或主动或被动地放弃了这个市场。作为消费者，我能做的实在有限。除了用自己的选择进行微弱的反馈之外，就只好希望各大公司用以人为本的态度对待电子产品吧。

最后，不妨回顾一些多年前的梗图，有的已经部分成为现实。

<img src='https://user-images.githubusercontent.com/27502578/105127905-d7216a00-5aaf-11eb-8b70-c4aea3deae03.png' width=350>
<br />
<img src='https://user-images.githubusercontent.com/27502578/105129351-db02bb80-5ab2-11eb-8df0-6867189e4365.png' width=350>
<br />
<img src='https://user-images.githubusercontent.com/27502578/105128430-05ec1000-5ab1-11eb-98c6-b99237382643.png' width=350>
<br />
<img src='https://user-images.githubusercontent.com/27502578/105128134-6a5a9f80-5ab0-11eb-8a73-42194540bdb6.png' width=350>
<img src='https://count.lnfinite.space/post/25.svg?plus=1' width='0' height='0' />

**Comment by jasonfan1997** [comment]
感覺和3nm製程一樣,手機屏幕的大小已經到物理極限了(再大就變pad了...),然而誰都沒有想到下一步要怎麼搞

**Comment by King-of-Infinite-Space** [comment]
硬件确实可能快到平台期了，不过软件还有不少优化的空间。

**Comment by King-of-Infinite-Space** [comment]
刚发布的小米12配备了6.28"屏幕，宽69.9mm。总算正常了，令人欣慰。
